src/uct/ib/base/ib_device.h

```diff
@@ -250,18 +251,32 @@ void uct_ib_device_cleanup(uct_ib_device_t *dev);
  */
 const uct_ib_device_spec_t* uct_ib_device_spec(uct_ib_device_t *dev);
 
+/**
+ * Select the best gid to use and set its information on the RoCE port -
+ * gid index, RoCE version and address family.
+ *
+ * @param [in] subnet_mask      Subnet mask, in bits (e.g. 24 for 1.2.3.*).
```

Comment:
subnet_mask -> prefix_length (in all places) since this is the size of the mask/prefix, not the mask itself

Reply:
sure